packages/blockchain-wallet-v4-frontend/src/modals/BuySell/UpgradeToGold/index.tsx

```diff
@@ -1,24 +1,16 @@
-import React, { PureComponent } from 'react'
+import React from 'react'
 import { connect, ConnectedProps } from 'react-redux'
-import { bindActionCreators, Dispatch } from 'redux'
+import { Dispatch } from 'redux'
 
 import { actions } from 'data'
 
 import Template from './template'
 
-export type LinkDispatchPropsType = {
-  buySellActions: typeof actions.components.buySell
-  verifyIdentity: () => void
-}
-
-class UpgradeToGold extends PureComponent<Props> {
-  render() {
-    return <Template {...this.props} />
-  }
+const UpgradeToGold: React.FC<Props> = (props: Props) => {
```

Comment:
since you're passing the Props to the React.FC already you don't need to do `: Props` in the big arrow func too.